### PR TITLE
Fix for #111 - correct source mapping when base is used

### PR DESCRIPTION
**Pull request by at0g**

### CODE REVIEW

**Comment by stevelacy** [comment]
Please add a test, or add to the current sourcemap test

File: test/main.js [hunk 1]
```diff
@@ -209,4 +209,29 @@ describe('gulp-stylus', function() {
 
 	});
 
+    it ('should use base relative paths when generating sourcemaps', function(done) {
+        var stream = sourcemaps.init();
+
+        var fakeFile = new gutil.File({
+            base: 'test',
```
Comment (at0g) [inline test/main.js]: ^ set base to **test** instead of **test/fixtures**

File: test/main.js [hunk 2]
```diff
@@ -209,4 +209,29 @@ describe('gulp-stylus', function() {
...
+        });
+
+        stream.write(fakeFile);
+        stream.pipe(stylus())
+            .on('error', done)
+            .on('data', function(newFile) {
+                should.exist(newFile);
+                newFile.sourceMap.sources.length.should.equal(1);
+                newFile.sourceMap.sources[0].should.equal('fixtures/normal.styl');
+                newFile.sourceMap.file.should.equal('fixtures/normal.css');
```
Comment (at0g) [inline test/main.js]: ^ Expect **fixtures/normal.css** instead of **normal.css**

**Comment by at0g** [comment]
Hello, have added as an additional test

**Comment by stevelacy** [comment]
Thanks

**Pull request merged**